src/Provider.js

```diff
@@ -1,8 +1,9 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import Context from './Context';
 import mapStateOnServer from './server';
 
+const defaultValue = {};
+export const Context = React.createContext(defaultValue);
 /* eslint-disable react/prop-types */
```

Comment:
can you move the comment before the 2 lines above, and put a new line between the comment and the const and the export?

```
/* eslint-disable react/prop-types */

const defaultValue = {};

export const Context = React.createContext(defaultValue);
```

After that, I'll merge this

Reply:
Updated.